### ISSUE TITLE
Built-in VoidWorld generator

### ISSUE THREAD

**Issue by Kydaix** [Imanity-Software/ImanitySpigot3-git#138]
### Is your feature request related to a problem?

Not really, I just think it could be a good way of doing it.

### Describe the solution you'd like.

Instead of having to install plugins in order to create empty worlds, I think having your own generator would be cooler! For example in the Bukkit server configuration.

### Describe alternatives you've considered.

I've installed plugin just to generate empty world using the Bukkit configuration file.

### Agreements

- [X] I have searched for and ensured there isn't already an open issue regarding this.
- [X] I have ensured the feature I'm requesting isn't already in the latest ImanitySpigot3.

**Comment by xpghard** [comment]
The generation of the map as a void is already built-in. All you need to do is change the server properies
level-type=void

**Comment by Kydaix** [comment]
No, this does not work for me.

**Comment by xpghard** [comment]
> No, this does not work for me. ![Capture](https://user-images.githubusercontent.com/90114872/183282650-2cd5625a-3a0a-4462-a942-be92c2936387.PNG)

because you use capslock ... the name must be in lower case

**Comment by Kydaix** [comment]
Let me try again! Thank you!

**Comment by Kydaix** [comment]
It works, thank you for your help!
